source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,67 @@
-# Architecture
+# Background FAQ
 
-There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts. Let's jump right in!
+[//]: # (If any commented statements become false, change prior text)
+
+There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts that are frequently asked questions. Let's jump right in!
 
 ## What are the components of the Bonsai Platform?
 
-The image below displays all of the components of Bonsai's platform. It enables  developers to more efficiently Build, Teach and Use intelligence models:
+This image displays all of the components of Bonsai's Platform. The Platform enables developers to more efficiently build, teach and use intelligence models:
 
 ![The Bonsai Platform][1]
 
-## Build
+###### Build
 
-* **CLI:** the Bonsai Command Line Interface (CLI) tool.
-* **brains.bons.ai:** The address for Bonsai's web interface; this is the browser based tool that includes your BRAIN Dashboard and BRAIN Details page.
-* **Simulators:** are a training source. They imitate a real world situation in a virtual environment. Simulators are often a complex, interactive environment. They usually have state, which is a representation of the world inside the virtual environment.
+* **Bonsai Command Line Interface (CLI):** is a command line tool that enables you to configure and control the Bonsai Artificial Intelligence (AI) Engine.
+* **beta.bons.ai:** The address for Bonsai's beta web interface; this is the browser based tool that includes your *BRAIN Dashboard* and *BRAIN Details* page.
+* **Simulators:** are currently the only training source implemented. Simulators are often a complex, interactive virtual environment. They usually have state, which is a representation of the world inside the virtual environment.
 * **Generators:** are a training source. They produce labeled data programmatically. This data is effectively infinite. A generator could, for example, produce a random (but known) integer, set of line segments, etc.
```

Comment:
Data isn't supported either yet, right? There is a note at the bottom of the section that again reiterates that simulators are the only thing you can currently use during private beta.